controllers/usersController.js

```diff
@@ -1,21 +1,36 @@
-const express = require('express');
-const { getAllUsers, getUserById } = require('../queries/usersQueries');
+const express = require("express");
+const { getAllUsers, getUserById } = require("../queries/usersQueries");
 
 const usersController = express.Router();
 
-usersController.get('/', (req, res) => {
-  users = getAllUsers() || [];
-  res.status(200).json({ data: users });
+usersController.get("/", async (req, res) => {
+  try {
+    users = (await getAllUsers()) || [];
+    if (users.length > 0) {
+      res.status(200).json({ data: users });
+      console.log("=== GET users: ", users, "===");
+    } else {
+      res.status(404).json({ error: `Users ${users} not found` });
+    }
+  } catch (err) {
+    res.status(500).json({ error: err.message });
+  }
 });
 
-usersController.get('/:id', (req, res) => {
-  const { id } = req.params;
-  user = getUserById(id);
-  if (user) {
-    res.status(200).json({ data: user });
-  } else {
-    res.status(404).json({ error: `User with id ${id} not found`});
+usersController.get("/:id", async (req, res) => {
+  try {
+    const { id } = req.params;
+    user = await getUserById(id);
+
+    if (user.length > 0) {
```

Comment:
I think this should probably be `if (user)` (not checking length) ... this is assuming that `getUserById` uses the `db.oneOrNone` method which returns either a single user or null.